src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,35 @@
+@import '/src/styles/blocks/variables';
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+
+  &__total {
+    display: flex;
+  }
+
+  &__number-of-reviews {
+    color: rgba(6, 11, 53, 1);
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 0;
+  }
+
+  &__total--n div:nth-child(-n + 4) {
+    background-image: url(/src/images/star-active.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+  }
```

Comment:
The selector `&__total--n div:nth-child(-n + 4)` is used to apply styles to the first four child elements. This approach is fine if you always want exactly four active stars, but it lacks flexibility. Consider using a more dynamic approach if the number of active stars might change, such as adding a class to the active stars or using JavaScript to dynamically apply styles based on data.